docs/api.md

```diff
@@ -292,6 +292,11 @@ content-server page.
     signed in email address will be used as the default.
   - If `action` is `force_auth`, the user is unable to modify the email
     address and is unable to sign up if the address is not registered.
+- `keys`: Optional. Boolean setting, set this if the relier wants access to the account encryption keys.
```

Comment:
IMO "wants access to the account encryption keys" suggests they can access the raw keys, which is not the case.  Something more generic like "wants to receive encryption keys" might be better.